scenarios/kill_wf_attempt/kill_wf_attempt.dig

```diff
@@ -0,0 +1,23 @@
+timezone: Asia/Tokyo
+
+
+_export:
+  baseurl: https://api-workflow.treasuredata.com/
```

Comment:
How about explicitly writing to change `baseurl` on README.md for customers using EU or Tokyo Region.

Reply:
Nice idea. I'll write comments to let users know that baseurl is up to account site.